ChemoTracker/components/PatientEducation/Search.js

```diff
@@ -0,0 +1,173 @@
+import React, { Component } from 'react';
+import { ActivityIndicator, FlatList, Linking, ListView, Text, TextInput, TouchableOpacity, View } from 'react-native';
+import { List, ListItem } from "react-native-elements";
+import { TabNavigator } from 'react-navigation';
+import { WebBrowser } from 'expo';
+import FontAwesomeIcon from 'react-native-vector-icons/FontAwesome';
+import IonIcon from 'react-native-vector-icons/Ionicons';
+import { searchStyles } from '../../styles/PatientEducation/patient-education.js';
+import color from '../../styles/color.js';
+
+const api = 'http://ec2-52-15-106-40.us-east-2.compute.amazonaws.com:8000';
```

Comment:
once my branch is merged, should move this into a separate file under `/services` folder.. you can see my latest changes in my branch on the folder and the file

Reply:
maybe make a file (e.g. constants, or strings, etc) with the api in there and import that so its hidden from implementation

Reply:
I'll make a file once Janice's branch is merged :)